### PR TITLE
Add support for dbt tasks

### PR DESCRIPTION
**Pull request by kzidane**
Add support for:
* Creating dbt tasks
* Overriding `dbt_commands` when calling `run_now`
* Running dbt job using `submit_run`

Tested manually and added some unit tests.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -17,4 +17,5 @@ venv
 .tox/
 tox.output
 .gitignore
-tox.ini
\ No newline at end of file
+tox.ini
+*.un~
```
Comment (KoningJasper) [inline .gitignore:21]: 🤔

File: tests/sdk/test_service.py [hunk 2]
```diff
@@ -130,6 +130,34 @@ def test_create_job(jobs_service):
...
 
+@provide_conf
+def test_create_dbt_task(jobs_service):
+    git_source  = {
+        'git_provider': 'github',
+        'git_url': 'https://github.com/foo/bar',
+        'git_branch': 'main'
+    }
+
+    tasks = {
```
Comment (KoningJasper) [inline tests/sdk/test_service.py:141]: Is this not an array/list?

File: .gitignore (hunk 1 above)
Comment (kzidane) [reply .gitignore:21]: Vim undo file 😬

File: tests/sdk/test_service.py (hunk 2 above)
Comment (kzidane) [reply tests/sdk/test_service.py]: It is. Fixed.

File: .gitignore (hunk 1 above)
Comment (pietern) [reply .gitignore:21]: This should really be in your global `.gitignore`..

There's poor precedent here though, so feel free to leave as is.

Comment (kzidane) [reply .gitignore]: @pietern reverted.

File: databricks_cli/version.py [hunk 3]
```diff
@@ -24,7 +24,7 @@
 import re
 
 
-version = '0.17.1.dev0' #  NOQA
+version = '0.17.2.dev0' #  NOQA
```
[the remark this answers is not in the document]
Comment (kzidane) [reply databricks_cli/version.py]: @pietern reverted.

**Comment by pietern** [comment]
@kzidane I regenerated `service.py` and that added the same fields to the repair API.

**Review by pietern — APPROVED** [review]

**Pull request merged**